core/src/main/java/org/kiwix/kiwixmobile/core/search/adapter/SearchListItem.kt

```diff
@@ -23,4 +23,5 @@ sealed class SearchListItem {
 
   data class RecentSearchListItem(override val value: String) : SearchListItem()
   data class ZimSearchResultListItem constructor(override val value: String) : SearchListItem()
+  data class ExactQueryPseudoSearchListItem(override val value: String) : SearchListItem()
```

Comment:
This isn't a SearchListItem and as far as I can see should not exist

Reply:
Removed